test/run_test.py

```diff
@@ -418,11 +418,11 @@ def run_test(
 
     # Can't call `python -m unittest test_*` here because it doesn't run code
     # in `if __name__ == '__main__': `. So call `python test_*.py` instead.
-    argv = [test_module + ".py"] + unittest_args
+    argv = [test_directory + "/" + test_module + ".py"] + unittest_args
```

Comment:
It would be nice to use `os.path.join(test_directory, test_module + ".py")` here... Or may be an f-string